### DISCUSSION TITLE
How to format a text value that defines a hyperlink to open in a new window?

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3524] [Q&A] [answered]
Hi,

I have in the search results a text value that if pasted into a browser will open a web page - so it has the data in a hyperlink but it is in the search results as text...

I can format this in the handle bars area so that it becomes clickable but I want it to open in a new tab and the target = blank html does not work for me - how can I get this to open in a new tab please?

The code in the handlebars area I have is:

<a href="{{0DE-MERLIN-DISPFORMLINKPA}}" target="_blank">View Document Properties</a>

If i only use this:

{{0DE-MERLIN-DISPFORMLINKPA}}

Then the value for the hyperlink is shown and it is not clickable - if i use the code line above then it is clickable but it does not respect the target="_blank" to open in a new tab.

Does anyone know how to do this please i.e. get it to open in a new tab?

Thanks!

**Comment by kasperbolarsen** [comment, marked as answer]
target="_blank" data-interception="off" ( in the handlebar )

:-)

**Reply by steveh2023** [reply]
Oh thanks very much - that worked!

**Reply by DanielSchoeneckGermany** [reply]
Thanks, that also solved my question on this :)

But I still have an issue with this:
The user clicks on a PDF document in the PNP search result
--> PDF document is opened in a new tab
--> the user closes the PDF document within the tab (not the tab itself)
--> in this tab the user is now in the document library where this PDF is stored.
--> the (not so experienced) user is confused because he was expecting to come back to the PnP Search results...

Question: is it possible that the click on the document in PnP just opens the document instead the library (and the document)

I attached a video for demonstration of the behaviour:

https://github.com/user-attachments/assets/ae282d37-5b35-4026-b8a5-af973e0fce64
